src/wp-includes/class-wp-image-editor-gd.php

```diff
@@ -261,13 +264,15 @@ public function multi_resize( $sizes ) {
 	 * Create an image sub-size and return the image meta data value for it.
 	 *
 	 * @since 5.3.0
+	 * @since 6.1.0 The $sizes parameter may now include a $name key for each entry.
 	 *
 	 * @param array $size_data {
 	 *     Array of size data.
 	 *
 	 *     @type int  $width  The maximum width in pixels.
 	 *     @type int  $height The maximum height in pixels.
 	 *     @type bool $crop   Whether to crop the image to exact dimensions.
+	 *     @type string $name Image size name.
```

Comment:
all the lines above should be realigned with the new prameter

Reply:
@adamsilverstein Good catch, updated!